docs/rules/g201_sql_injection_via_format_string.md

```diff
@@ -0,0 +1,67 @@
+---
+id: g201
+title: G201: SQL query construction using format stringg
+---
+
+SQL injection is one of the top security issues develops made and the consequences of this can be enormous. 
```

Comment:
I would  change 'develops made' to 'developers make', and maybe 'enormous' to 'severe'.

Reply:
I agree and I will change it.